pkg/registry/apis/dashboard/sub_dto.go

```diff
@@ -164,35 +121,84 @@ func (r *DTOConnector) Connect(ctx context.Context, name string, opts runtime.Ob
 		}
 	}
 
-	access.Slug = slugify.Slugify(dash.Spec.GetNestedString("title"))
-	access.Url = dashboards.GetDashboardFolderURL(false, name, access.Slug)
-
 	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		responder.Object(http.StatusOK, &dashboard.DashboardWithAccessInfo{
-			Dashboard: *dash,
-			Access:    access,
-		})
+		// Skip the access info and return the dashboard that may be loaded with large object support
+		if req.URL.Query().Get("includeAccess") == "false" {
```

Comment:
@suntala -- looks like changing this to skip filling in the access info is straight forward.  The challenge for your use case is that the standard k8s client-go does not let you pass parameters! so you would need to use standard HTTP client (not hard or bad)